setup.cfg

```diff
@@ -31,7 +31,7 @@ install_requires =
 	janus
 	ansible-runner
 	websockets
-	drools_jpy == 0.3.8
+	drools_jpy == 0.3.9
```

Comment:
Is there some feature or bugfix coming from drools that we might want to reflect in the changelog?